packages/frames.js/src/core/errors.ts

```diff
@@ -9,3 +9,20 @@ export class InvalidFrameActionPayloadError extends Error {
     super("Invalid frame action payload");
   }
 }
+
+export class FrameMessageError extends Error {
+  status: number;
+
+  /**
+   *
+   * @param message - Message to show the user (up to 90 characters)
+   * @param status - 4XX status code
+   */
+  constructor(message: string, status: number) {
+    if (message.length > 90) throw new Error("Message too long");
+    if (status < 400 || status >= 500) throw new Error("Invalid status code");
```

Comment:
we don't allow 500?

Reply:
not allowed by the spec

https://warpcast.notion.site/Frames-Errors-ddc965b097d44d9ea03ddf98498597c6

Reply:
ah ok, still weird decision but I understand that this is for UI feedback as well.

Reply:
It is little confusing because people could use it to return 500. Is that a way to disallow 500 at least on type level?

Reply:
can do